app/models/concerns/orchestration.rb

```diff
@@ -172,6 +172,7 @@ def add_conflict(e)
 
   def execute(opts = {})
     obj, met, param = opts[:action]
+    params = opts[:params]
```

Comment:
041436c8e2eb9528a7f7c1c760621d1639100533 already added a way to pass parameters to orchestration queue. I think, we should stick with only one solution. I don't care which one, though.